FlowCrypt/Controllers/SideMenu/NavigationController/MainNavigationController.swift

```diff
@@ -82,3 +79,13 @@ extension UINavigationController {
         }
     }
 }
+
+extension UIBarButtonItem {
+    static func defaultBackButton(with action: (() -> Void)? = nil) -> NavigationBarActionButton {
+        NavigationBarActionButton(
+            UIImage(named: "arrow-left-c"),
+            action: action,
+            accessibilityIdentifier: "aid-back-icon"
```

Comment:
Maybe it'll be better to call this button `aid-back-button`? As `back-icon` sounds like something non-interactive, just icon

Reply:
no objections. 
Changed according to https://github.com/FlowCrypt/flowcrypt-ios/pull/1217#discussion_r767956036